webapi/src/main/scala/org/knora/webapi/responders/admin/AssetPermissionResponder.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.responders.admin
+
+import zio.*
+
+import dsp.errors.InconsistentRepositoryDataException
+import dsp.errors.NotFoundException
+import org.knora.webapi.messages.SmartIri
+import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM.ShortcodeIdentifier
+import org.knora.webapi.messages.admin.responder.sipimessages.PermissionCodeAndProjectRestrictedViewSettings
+import org.knora.webapi.messages.store.triplestoremessages.IriSubjectV2
+import org.knora.webapi.messages.store.triplestoremessages.LiteralV2
+import org.knora.webapi.messages.store.triplestoremessages.SparqlExtendedConstructResponse
+import org.knora.webapi.messages.twirl.queries.sparql
+import org.knora.webapi.messages.util.PermissionUtilADM
+import org.knora.webapi.slice.admin.domain.model.User
+import org.knora.webapi.store.triplestore.api.TriplestoreService
+import org.knora.webapi.store.triplestore.api.TriplestoreService.Queries.Construct
+
+/**
+ * Responds to requests for information about binary representations of resources, and returns responses in Knora API
+ * ADM format.
+ */
+final case class AssetPermissionResponder(
```

Comment:
Following existing convention, shouldn't the class be named somehow in plural: `AssetPermissionsResponder`, `AssetsPermissionsResponder` or `AssetsPermissionResponder`?  I'm not sure which one is grammatically correct or fits better. @BalduinLandolt

Reply:
I will rename it to `AssetsPermissionsResponder` in a followup PR.

Reply:
=> https://github.com/dasch-swiss/dsp-api/pull/2996/commits/cbc2109b4737d7cc67d79ea835ea6bc87eda347a